components/st-microelectronics/STM32F031F6P6-supports.stanza

```diff
@@ -0,0 +1,155 @@
+defpackage STM32F031F6P6-supports :
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+
+  import ocdb/defaults
+  import ocdb/bundles
+
+; ; Autogenerated by ocdb/scripts/cubemx-importer
+; Family: STM32L0, Package: TSSOP20
+; STM32L031F(4-6)Px
+
+public defn stm-support ( ) :
+  inside pcb-component :
+
+    supports swd:
+      swd.swdio  => self.PA[13]
+      swd.swdclk => self.PA[14]
+
+    supports gpio:
+      gpio.gpio => self.PC[14]
+
+    supports gpio:
+      gpio.gpio => self.PC[15]
+
+    supports gpio:
+      gpio.gpio => self.PA[0]
+
+    supports gpio:
+      gpio.gpio => self.PA[1]
+
+    supports gpio:
+      gpio.gpio => self.PA[2]
+
+    supports gpio:
+      gpio.gpio => self.PA[3]
+
+    supports gpio:
+      gpio.gpio => self.PA[4]
+
+    supports gpio:
+      gpio.gpio => self.PA[5]
+
+    supports gpio:
+      gpio.gpio => self.PA[6]
+
+    supports gpio:
+      gpio.gpio => self.PA[7]
+
+    supports gpio:
+      gpio.gpio => self.PB[1]
+
+    supports gpio:
+      gpio.gpio => self.PA[9]
+
+    supports gpio:
+      gpio.gpio => self.PA[10]
+
+    supports gpio:
+      gpio.gpio => self.PA[13]
+
+    supports gpio:
+      gpio.gpio => self.PA[14]
```

Comment:
is there a way to have a for loop for all these supports gpio?

Reply:
No. This is autogenerated code.